build/shade/_k-standard-goals.shade

```diff
@@ -219,6 +219,10 @@ default PACKAGELIST_JSON_FILENAME = 'NuGetPackageVerifier.json'
         var project = (JsonObject)Json.Deserialize(projectText);
         var configs = project.ValueAsJsonObject("frameworks");
         var targetFrameworks = configs == null ? new string[0] : configs.Keys;
+        if (!string.IsNullOrEmpty(BuildFramework))
+        {
+            targetFrameworks = targetFrameworks.Where(t => t.Equals(BuildFramework, StringComparison.OrdinalIgnoreCase)).ToList();
```

Comment:
> In what scenario would we have multiple matches?

We won't have multiple matches now. If we want to support more than one framework by having the env var to contain comma separated values. But I don't think it is very useful.

> `ToList` is not required?

`ToList` is required because `targetFrameworks` is of type `ICollection` and `Where` returns `IEnumerable`.